templates/publisher/publicise.html

```diff
@@ -0,0 +1,156 @@
+{% extends "publisher/_publisher_layout.html" %}
+
+{% block meta_title %}
+Publicise {% if display_title %}{{ display_title }}{% else %}{{ snap_title }}{% endif %}
+{% endblock %}
+
+{% block content %}
+<div id="main-content">
+  {% set selected_tab='publicise' %}
+  {% include "publisher/_header.html" %}
+
+  <div class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-2">
+        <label>Language:</label>
+      </div>
+      <div class="col-8">
+        <form class="p-form--inline js-language-select">
+          <select name="language">
+            {% for lang, data in available.items() %}
+              <option value="{{ lang }}">{{ data.title }}</option>
+            {% endfor %}
+          </select>
+        </form>
+        <p class="u-no-margin--bottom">You can contribute to translate these assets <a href="https://github.com/snapcore/snap-store-badges" target="_blank" class="p-link--external">in this repository</a>.</p>
+      </div>
+    </div>
+`
+    <div class="row">
+      <hr />
+    </div>
+
+    {% for lang, data in available.items() %}
+      <div id="{{ lang }}_content" class="js-language-content{% if loop.index != 1 %} u-hide{% endif %}">
+        <div class="row">
+          <div class="col-2">
+          </div>
+          <div class="col-8">
+            <div class="col-5">
+              <p class="snapcraft-publicise__images">
+                {% set url = "images/badges/" + lang + "/snap-store-black.svg" %}
+                <a href="https://snapcraft.io/{{ snap_name }}" title="{{ data.text }}">
+                  <img src="{{ static_url(url) }}" />
+                </a>
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-2">
+            <label>HTML:</label>
+          </div>
+          <div class="col-8">
+            <pre><code>&lt;a href="https://snapcraft.io/{{ snap_name }}" title="{{ data.text }}"&gt;
+  &lt;img src="https://snapcraft.io/static/images/badges/{{ lang }}/snap-store-black.svg" alt="" /&gt;
```

Comment:
Why moving the images into snapcraft.io app? Just to make URLs nicer?
I'm bit worried that this would mean some unnecessary asset traffic will hit the server. 

Also it seems that cache is set to 12h (I guess default for our assets), but these badges are (almost) never going to change, we could safely cache them for a year. Especially that our static URLs have a version hash, so would update when file changes.

Reply:
As the cache limit is not directly related to this PR I'm fine with fixing that separately, but it would be good to create issue for that to have it tracked.